### ISSUE TITLE
FileNotFoundException on directory drag and drop, SCIFIO on

### ISSUE THREAD

**Issue by karlduderstadt** [imagej/imagej-legacy#284]
I discovered that I get a FileNotFoundException when dragging and dropping a directory with SCIFIO turned on. After the exception comes up the Image Sequence dialog also appears. With SCIFIO turned off, I do not get an exception.
<img width="1004" alt="Screenshot 2022-09-22 at 08 30 11" src="https://user-images.githubusercontent.com/17951396/191676422-ff395e08-0013-4a00-bd51-2478a2e68d87.png">

**Comment by ctrueden** [comment]
This issue happens due to a bug in SciJava Common's `TextService`, which was recently fixed: scijava/scijava-common@aa7fe2e241c9e5dcca2ffad7945475aca47dc72c. I am releasing scijava-common 2.89.1 including this fix now, and will upload it to the core update site later today.

**Comment by ksugar** [comment]
@ctrueden Great!!! Using the fixed version, I could perform drag-and-drop of a directory with SCIFIO turned on without giving this error. I noticed that a *directory* with a name ending `.txt` (it's unusual) still gives this error. Is it the expected behavior?

**Comment by ksugar** [comment]
It seems that it's using `TableIOPlugin`.

```
[ERROR] java.io.FileNotFoundException: /home/user/directory.txt (Is a directory)
	at java.io.RandomAccessFile.open0(Native Method)
	at java.io.RandomAccessFile.open(RandomAccessFile.java:316)
	at java.io.RandomAccessFile.<init>(RandomAccessFile.java:243)
	at org.scijava.io.handle.FileHandle.initRAF(FileHandle.java:317)
	at org.scijava.io.handle.FileHandle.reader(FileHandle.java:298)
	at org.scijava.io.handle.FileHandle.length(FileHandle.java:112)
	at org.scijava.table.DefaultTableIOPlugin.open(DefaultTableIOPlugin.java:171)
	at org.scijava.table.DefaultTableIOPlugin.open(DefaultTableIOPlugin.java:158)
	at org.scijava.table.DefaultTableIOPlugin.open(DefaultTableIOPlugin.java:61)
	at org.scijava.table.io.TableIOPlugin.open(TableIOPlugin.java:48)
	at org.scijava.table.io.TableIOPlugin.open(TableIOPlugin.java:44)
	at org.scijava.io.AbstractIOPlugin.open(AbstractIOPlugin.java:82)
	at net.imagej.legacy.plugin.DefaultLegacyOpener.open(DefaultLegacyOpener.java:134)
	at net.imagej.legacy.DefaultLegacyHooks.interceptDragAndDropFile(DefaultLegacyHooks.java:417)
	at ij.plugin.DragAndDrop.openFile(DragAndDrop.java)
	at ij.plugin.DragAndDrop.run(DragAndDrop.java:160)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by ctrueden** [comment]
@ksugar Thanks for the report about the table plugin. I fixed it ([scijava/scijava-table@f3b17b78](https://github.com/scijava/scijava-table/commit/f3b17b78f619bc9fb298ed0c0102c5d813aa761e)) and released scijava-table 1.0.0. The scijava-plugins-io-table component is now retired, merged with scijava-table. I pushed the update to the core update site. Let me know how it works for you.
